tests/google/cloud/hooks/test_dataproc.py

```diff
@@ -11,15 +11,15 @@
 from astronomer.providers.google.cloud.hooks.dataproc import DataprocHookAsync
 
 
-@mock.patch("airflow.providers.google.common.hooks.base_google.GoogleBaseHook._get_credentials")
+@mock.patch("airflow.providers.google.common.hooks.base_google.GoogleBaseHook.get_credentials")
```

Comment:
Won't the tests fail for previous versions which still have the method name as `_get_credentials`?

Reply:
In CI there will be one provider version if I use >= 8.40 I need `get_credentials` to run.
if CI has <8.4.0 then I need `_get_credentials` to work. I need to mock credentials but for which version?